formfunction.html

```diff
@@ -43,9 +43,9 @@
                 <p class="no-above">
                     Students can choose to attend one of our three sessions. The sessions are:
                     <ul class="no-above2">
-                        <li>5:00 to 6:00pm PT every Wednesday</li>
-                        <li>4:45 to 5:45pm PT every Thursday</li>
-                        <li>4:45 to 5:45pm PT every Friday</li>
+                        <li>Every Monday from 4:30 to 5:30 pm PT / 6:30 to 7:30 pm CT / 8:30 to 9:30 pm ET</li>
+                        <li>Every Wednesday from 4:30 to 5:30 pm PT / 6:30 to 7:30 pm CT / 8:30 to 9:30 pm ET</li>
+                        <li>Every Friday from 4:30 to 5:30 pm PT / 6:30 to 7:30 pm CT / 8:30 to 9:30 pm ET</li>
```

Comment:
Since it's 4:30-5:30pm PT, I think here for the eastern time it should be from 7:30 to 8:30pm instead.